src/components/MultiHopTrade/components/TradeInput/components/MaxSlippage.tsx

```diff
@@ -160,18 +162,20 @@ export const MaxSlippage: React.FC<MaxSlippageProps> = ({
     isAmountPositive,
     isLoading,
     minAmountAfterSlippageTranslation,
-    shouldShowSlippageMinAmount,
+    isUnknownAutoSlippage,
     symbol,
     translate,
   ])
 
   return (
-    <Row Tooltipbody={tooltipBody}>
+    <Row Tooltipbody={!isUnknownAutoSlippage ? tooltipBody : undefined}>
```

Comment:
No slippage, no tooltip to say "Minimum after slippage: <expected after slippage>" nor "Your transaction will revert if the price changes unfavorably by more than this percentage." (there is no known percentage)